pmd-core/src/test/java/net/sourceforge/pmd/FooRule.java

```diff
@@ -18,6 +18,7 @@
     public FooRule() {
         setLanguage(LanguageRegistry.getLanguage(DummyLanguageModule.NAME));
         setName("Foo");
+        setDescription("Description with Unicode Character U+2013: – .");
```

Comment:
Maybe we can make this even clearer
```suggestion
        setDescription("Description with Unicode Character U+2013: \u2013 .");
```